src/components/DateTimePicker/Internal/OcPicker.tsx

```diff
@@ -83,18 +86,23 @@ function InnerPicker<DateType>(props: OcPickerProps<DateType>) {
     popupPlacement,
     popupStyle,
     shape = DatePickerShape.Rectangle,
+    showNow,
```

Comment:
what is this? Its not jumping out at me in storybook

Reply:
The Now button when showTime is true.